blueprints/controller/module-unification-files/__root__/__path__/controller.js

```diff
@@ -0,0 +1,4 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+});
```

Comment:
this looks like the same content as the existing https://github.com/dexturr/ember.js/blob/d23b14d99095c062693193e447b0ca578006e063/blueprints/controller/files/__root__/__path__/__name__.js. Perhaps we can use that for both and delete this file?

Reply:
Yep, was copying the pattern of components but this is unnecessary here. Have done this now 👍